content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Running Kubernetes node components as a non-root user (`KubeletInUserNamespace`)
```

Comment:
we don't have to mention the gate name in the title

Reply:
fixed